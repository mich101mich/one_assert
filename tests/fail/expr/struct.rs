fn main() {
    struct S {
        b: bool,
    }
    one_assert::assert!(S { b: true });
}
