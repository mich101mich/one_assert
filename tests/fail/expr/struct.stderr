error: Expected a boolean expression, found a struct literal
 --> tests/fail/expr/struct.rs:5:25
  |
5 |     one_assert::assert!(S { b: true });
  |                         ^^^^^^^^^^^^^
